parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1800,11 +1800,8 @@ applySelection hqs file = \sr@SlurpResult{..} ->
   selection = SlurpComponent selectedTypes selectedTerms
   closed = SC.closeWithDependencies file selection
   selectedTypes, selectedTerms :: Set v
-  selectedTypes = Set.map var $ R.dom (Names.types selectedNames0)
-  selectedTerms = Set.map var $ R.dom (Names.terms selectedNames0)
-
-var :: Var v => Name -> v
-var name = Var.named (Name.toText name)
+  selectedTypes = Set.map Name.toVar $ R.dom (Names.types selectedNames0)
+  selectedTerms = Set.map Name.toVar $ R.dom (Names.terms selectedNames0)
```

Comment:
This code isn't actually being used (although it might be in the future, if there is demand for being able to add individual or few definitions from a .u file, rather than slurping them all at once.)  In other words, I think line 1790 is always hit, and not 1791.

Reply:
Moreover, it looks like these are only dealing with `Name` because `UF.typecheckedToNames0` uses `Name` because `Names.filterByHQs` uses `Name`.  However, I don't think there's anything about the filtering that necessarily involves `Name`.

Reply:
> This code isn't actually being used (although it might be in the future, if there is demand for being able to add individual or few definitions from a .u file, rather than slurping them all at once.)

Can't you apply a filter with e.g. `add <term>`?

Reply:
Oops, you sure can.  I was thinking it wasn't fully implemented.